style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  margin-top: 10px;
+}
+
+body {
+  position: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
+
+.toolbar {
+  left: 0;
+  right: 0;
+  top: 40px;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] :recycle: Kindly make sure you stick to the figma design as much as possible, After making the changes above, I suggest to add `flex-direction:row`,justify-content :space-between` :

  - Your design:
![your Header](https://user-images.githubusercontent.com/97834160/206290091-a282b6be-5b80-48b3-b0b1-6a6231cc6e32.png)

  - Figma design:
![figHeader](https://user-images.githubusercontent.com/97834160/206290037-3887ee5a-122a-4f02-ae89-0e7b2bc201e4.png)